src/processTargets/index.ts

```diff
@@ -142,6 +155,51 @@ function processRangeTarget(
   );
 }
 
+function processColumnTarget(
+  target: RangeTarget,
+  anchorTarget: TypedSelection,
+  activeTarget: TypedSelection,
+  isForward: boolean
+) {
+  const anchorLine = targetToLineLimitPosition(
+    anchorTarget,
+    isForward,
+    !target.excludeAnchor
+  );
+  const activeLine = targetToLineLimitPosition(
+    activeTarget,
+    !isForward,
+    !target.excludeActive
+  );
+  const anchorSelection = anchorTarget.selection.selection;
+  const delta = isForward ? 1 : -1;
+  const results: TypedSelection[] = [];
+
+  for (let i = anchorLine; true; i += delta) {
```

Comment:
This would be a bit cleaner imo with a loadash range + map but not worth rewriting; maybe something to play with next time

Reply:
I must admit that I'm not a big fan of lodash. But you're probably right in this case and I did think about it.